osdep/terminal-unix.c

```diff
@@ -492,8 +492,8 @@ void terminal_setup_getch(struct input_ctx *ictx)
     }
 
     setsigaction(SIGINT,  quit_request_sighandler, SA_RESETHAND, false);
-    setsigaction(SIGQUIT, quit_request_sighandler, SA_RESETHAND, false);
-    setsigaction(SIGTERM, quit_request_sighandler, SA_RESETHAND, false);
+    setsigaction(SIGQUIT, quit_request_sighandler, 0, false);
+    setsigaction(SIGTERM, quit_request_sighandler, 0, false);
```

Comment:
I'd only do it for `SIGTERM` given the `timeout` situation. While I don't see any issues with removing RESETHAND for `SIGQUIT` I don't think it's worth changing unless an actual use-case for it shows up.

Also regarding the commit message:

> This change is because it was too easy to send multiple quit requests
accidentally

This should be removed since it's not "accidental".

> before mpv is prepared to handle these signals during
preinit

This is misleading because if a signal arrives (single or double) before we installed the signal handlers then this patch makes no difference.

Reply:
Applied the commit messages changes. I don't think it's worth having different behavior for the same type of signals, so I'll keep the SIGQUIT change